### PR TITLE
Snake Game Evolutionary AI

### PR DESCRIPTION
**Pull request by deplodog**
### Community Contribution

This is evolutionary AI snake game, based on idea of snake game + AI from this video and your flappy-bird neural network. 

**Link to live project, video, or image:**
demo: https://editor.p5js.org/deplodog@gmail.com/present/HrbT-gueC
demo how i trained it: https://editor.p5js.org/deplodog@gmail.com/present/JiFD0311f
source: https://editor.p5js.org/deplodog@gmail.com/sketches/JiFD0311f

### Sharing

- [x]   I would be happy to have my project shared on The Coding Train's social media!

**Twitter:** https://twitter.com/deplodog

### CODE REVIEW

**Comment by gruselhaus** [comment]
wow! this is awesome!

**Comment by deplodog** [comment]
Thanks. I'm trying to figure out how to make pull request works, and its seems like i've got some problems here. 
"Jest / build (10.x) (pull_request) Failing after 1m"
Should i fix it somehow?

File: _TeachableMachine/2-teachable-game.md [hunk 1]
```diff
@@ -49,6 +49,12 @@ contributions:
       url: "https://simontiger.com"
     url: "https://editor.p5js.org/simontiger/present/qH_ZSvup5"
     source: "https://editor.p5js.org/simontiger/sketches/qH_ZSvup5"
+    - title: "Snake Game Evolutionary AI"
```
Comment (mikakruschel) [inline _TeachableMachine/2-teachable-game.md]: This line is not correctly intended

**Comment by deplodog** [comment]
Thank you, guys. Fixed it

File: _TeachableMachine/2-teachable-game.md (hunk 1 above)
Comment (gruselhaus) [reply _TeachableMachine/2-teachable-game.md]: Thanks @mikakruschel!

**Pull request merged**